demo/genome_Snakefile.smk

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env snakemake
+
+'''
+It will only run through the workflow(eg. genome) sub directory.
+'''
+##### SETUP #####
+
+import oncopipe as op
+
+SAMPLES = op.load_samples("data/samples.tsv")
+GENOME = op.filter_samples(SAMPLES, seq_type = "genome")
+
+
+##### REFERENCE_FILES WORKFLOW #####
+
+
+subworkflow reference_files:
+    workdir:
+        "reference/"
+    snakefile:
+        "../workflows/reference_files/2.4/reference_files.smk"
+    configfile:
+        "../workflows/reference_files/2.4/config/default.yaml"
+
+
+##### CONFIGURATION FILES #####
+
+
+
+# Load module-specific configuration
+
+configfile: "../modules/slms_3/1.0/config/default.yaml"
+configfile: "../modules/sage/1.0/config/default.yaml"
+configfile: "../modules/manta/2.3/config/default.yaml"
+configfile: "../modules/gridss/1.1/config/default.yaml"
+configfile: "../modules/strelka/1.1/config/default.yaml"
+configfile: "../modules/lofreq/1.0/config/default.yaml"
+configfile: "../modules/starfish/2.0/config/default.yaml"
+configfile: "../modules/controlfreec/1.1/config/default.yaml"
+configfile: "../modules/utils/2.1/config/default.yaml"
+configfile: "../modules/picard_qc/1.0/config/default.yaml"
+configfile: "../modules/bam2fastq/1.2/config/default.yaml"
+#configfile: "../modules/vcf2maf/1.2/config/default.yaml"
```

Comment:
When you leave lines commented out you should follow them with a comment explaining why they are there. Is this a line they should uncomment if they have pre-installed vcf2maf? If so, you should tell them that with a comment.